packages/cli/src/platform/project/$.ts

```diff
@@ -8,8 +10,8 @@ export class $ implements Command {
   private constructor(private readonly commands: Commands) {}
 
   public async parse(argv: string[]) {
-    await Promise.resolve('todo')
-
-    return JSON.stringify(argv)
+    const args = arg(argv, {})
+    if (isError(args)) throw args
```

Comment:
What we normally do in the CLI is to return the help message along (you can check for the `public help` function definition of any command)
```suggestion
    if (isError(args)) return this.help(args.message)
```

Reply:
Can you confirm this is correct:

1. Throw feature flag error
2. Return all other errors

Reply:
all Errors are thrown, here you can check for the public help function definition of any command like I said earlier

Reply:
Maybe I took a shortcut / misunderstood something by the way. Note that this is probably interesting for you 
See https://github.com/prisma/prisma/blob/ce9bb031140b994e04931d80847b6f04e745e860/packages/cli/src/bin.ts#L115-L127
If a command returns a `HelpError` then we print the message.
If it's another error, it prints the full error, which might not be what you want (you probably only want to print the error message as well?)

Reply:
> all Errors are thrown, here you can check for the public help function definition of any command like I said earlier

@Jolg42 I'm confused

https://github.com/prisma/prisma/pull/22041#discussion_r1400238956

Reply:
Will resolve for now, let's keep discussing though.